internal/backends/qt/qt_window.rs

```diff
@@ -1612,6 +1612,17 @@ impl WindowAdapterSealed for QtWindow {
         self.tree_structure_changed.replace(true);
     }
 
+    fn register_root_component(&self, window_item: Pin<&i_slint_core::items::WindowItem>) {
+        let default_font_size_prop =
+            i_slint_core::items::WindowItem::FIELD_OFFSETS.default_font_size.apply_pin(window_item);
+        if default_font_size_prop.get().get() <= 0 as Coord {
+            let default_font_size = cpp!(unsafe[] -> i32 as "int" {
+                return QFontInfo(qApp->font()).pixelSize();
+            });
+            default_font_size_prop.set(LogicalLength::new(default_font_size as f32));
```

Comment:
Ideally that should be a binding to a property that would be updated in QWidget::changeEvent  (so it will be changed when the font are changed)
That said, this is only for the non-native style  (since this is otherwise taken care of by the NativeStyleMetrics

Reply:
Yes. Ahh, that is the case I didn't consider. I'll add a comment here at least. Thanks :)